contrib/systemd/user_network.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=vsock_network
+After=NetworkManager.service
```

Comment:
I'd add `After=sys-devices-virtual-net-vsock0.device`, and possibly `BindsTo=sys-devices-virtual-net-vsock0.device`,  but I think with the latter Praveen had issues with `systemctl restart user_network.service`

Reply:
well, i know this works as is ... do you want to change it when you do a switch over?